### DISCUSSION TITLE
Documentation changes

### DISCUSSION THREAD

**Discussion by Akhil-CM** [zimfw/zimfw#452] [General]
I would like to suggest changes to the README to include information on `${ZIM_HOME}` and also an example `.zimrc` with some common modules:
[Example](https://github.com/zdharma-continuum/zinit#plugins-and-snippets) from [zinit](https://github.com/zdharma-continuum/zinit)
[Example](https://github.com/agkozak/zcomet#sample-zshrc) from [zcomet](https://github.com/agkozak/zcomet)

The main issue is with calling `compinit` which has special order as compared to others.

**Comment by ericbn** [comment]
Great idea. Documentation improvements are more than welcome! (We're even still missing having a CONTRIBUTING.md file)

Did you think on the specific details about the `${ZIM_HOME}` and `.zimrc` documentation changes you would like see? You can propose those on a pull request, and then we can continue the discussion from there.

Hope it's clear that you should not call `compinit` if you're already using `zmodule completion` in your `.zimrc`, because that module does that for you. Otherwise, it's one more thing to improve regarding the documentation.  ;- )

**Reply by Akhil-CM** [reply]
1) Most people would like to keep their config files in `~/.config` like in `~/.config/zsh`. I think zim defaults to `$ZDOTDIR/zim` so that will be inherently taken care of.

> you should not call compinit if you're already using zmodule completion in your .zimrc, because that module does that for you.

This is exactly the type of information I was looking for. I will try to make a PR

**Comment by RayZ0rr** [comment]
I was also thinking about the same. I would like to work on a PR for documentation additions.

**Comment by ericbn** [comment]
Maybe we can even stop using the automatic installer, that adds a predefined Zim configuration, in favor of having a good documentation that will allow users to start with their own custom configuration from the start.

**Reply by RayZ0rr** [reply]
That's also an option. For the time being, I would keep both options available. Anyway, I'll open a PR on documentation soon.
